servers/zms/src/main/java/com/yahoo/athenz/zms/store/ObjectStore.java

```diff
@@ -41,8 +41,9 @@ public interface ObjectStore {
      * @param domainLimit domain tags limit
      * @param roleLimit role tags limit
      * @param groupLimit group tags limit
+     * @param policyLimit policy tags limit
      */
-    void setTagLimit(int domainLimit, int roleLimit, int groupLimit);
+    void setTagLimit(int domainLimit, int roleLimit, int groupLimit, int policyLimit);
```

Comment:
this api is also modified in the service tag so it's not going to cause a failure with a merge